### PR TITLE
[FIX]  Re-Generated Testnet Genesis Keys

### PR DESCRIPTION
**Pull request by yourbuddyconner**
The test keys in use needed to be regenerated in Base58 encoding.

### CODE REVIEW

File: ansible/coda_start.sh.j2 [hunk 1]
```diff
@@ -114,7 +114,7 @@ case $CODA_ROLE in
     sleep 10 # wait for daemon to come up
     export OMP_NUM_THREADS=8
     while [ "$(pgrep  -f 'coda internal snark-worker' | wc -l)" -lt "$CORE_SCALE" ]; do
-        nice start_worker {{ snark_leader_ip }}:8301
+        start_worker {{ snark_leader_ip }}:8301
```
Comment (bkase) [inline ansible/coda_start.sh.j2]: Why remove nice?

File: ansible/testkeys/keyprep.py [hunk 2]
```diff
@@ -1,35 +0,0 @@
-#!/usr/bin/env python3
```
Comment (bkase) [inline ansible/testkeys/keyprep.py]: Should we keep this script?

Comment (yourbuddyconner) [reply ansible/testkeys/keyprep.py]: There's an identical script in the coda repo that does this magic, so I don't think so.

File: ansible/coda_start.sh.j2 (hunk 1 above)
Comment (yourbuddyconner) [reply ansible/coda_start.sh.j2]: It wasn't working with a function as an argument, so I removed it.

**Pull request merged**